src/lib.rs

```diff
@@ -218,6 +243,35 @@ impl HidApi {
         }
     }
 
+    /// Open a HID device using libusb_wrap_sys_device.
+    pub fn wrap_sys_device(&self, sys_dev: isize, interface_num: i32) -> HidResult<HidDevice> {
+        if cfg!(not(any(
+            feature = "linux-static-libusb",
+            feature = "linux-shared-libusb"
+        ))) {
+            return Err(HidError::HidApiError {
+                message: String::from("wrap_sys_device is not implement"),
+            });
+        }
+
+        let device = unsafe {
+            ffi::hid_libusb_wrap_sys_device(sys_dev, interface_num)
```

Comment:
does not actually compile successful on platforms other than libusb based ones. The `ffi::hid_libusb_wrap_sys_device()` only exists on those. Compile this whole function conditionally, to solve the problem.